scripts/github/build/bumpversion.sh

```diff
@@ -4,7 +4,7 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 cd $DIR/../../../
 BRANCH=${GITHUB_REF##*/}
-if [[ $BRANCH =~ ^develop(-patch.*)?$ ]]
+if [[ ! $BRANCH =~ ^master(-patch.*)?$ ]]
```

Comment:
```suggestion
if [[ ! $GITHUB_REF_NAME =~ ^master(-patch.*)?$ ]]
```

not sure what's the intention here, but if you want to just have the alpha handling for workflow_dispatch you can use also $GITHUB_EVENT_NAME here